projects/element-ng/status-counter/si-status-counter.module.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { NgModule } from '@angular/core';
+
+import { SiStatusCounterComponent } from './si-status-counter.component';
+
+@NgModule({
```

Comment:
I think you can delete this file, we moved to standalone, so don't need this anymore. If they still want the module, they can use the alias.